skimage/measure/_regionprops.py

```diff
@@ -1,14 +1,15 @@
 from math import sqrt, atan2, pi as PI
 import numpy as np
 from scipy import ndimage as ndi
+from skimage.util import pad
 
 from ._label import label
 from . import _moments
 
 from functools import wraps
 
 
-__all__ = ['regionprops', 'perimeter']
+__all__ = ['regionprops', 'euler_number', 'perimeter', 'crofton_perimeter']
```

Comment:
I'd rename this to `perimeter_crofton` to help with autocompletion — users will then be able to easily see what their options are when searching for `perimeter`

Reply:
done